### ISSUE TITLE
feat: add standard icon association for localization/language files

### ISSUE THREAD

**Issue by SethFalco** [microsoft/vscode#184358]
Similar to how most image formats (WEBP, AVIF, PNG, JPG, GIF, etc) have an icon association to depict that they are images, could there please be a conventional icon for localization files?

By default, this icon would be associated with formats like:

* XLIFF (`*.xlf`)
* TMX (`*.tmx`)
* Fluent (`*.ftl`)
* PO (`*.pot` / `*.po`)
* etc

Some extensions may choose to reuse the icon for custom formats. Other broader file formats (like `*.json`) that extensions know to be a language file in the context of the libraries/stack used may use the icon as well.

### Related

* Originally discussed in https://github.com/projectfluent/fluent/issues/356

**Comment by daviddossett** [comment]
I think this is something that would have to be added to https://github.com/jesseweed/seti-ui, the default file icon font used by VS Code. From there we may have to do more work to actually render it where appropriate. @isidorn may have more details there since we've talked about new icons in Seti before.

**Comment by isidorn** [comment]
Yes, I think that is all there is to it. Adding to the seti built-in theme.
But I would not add this right away, I suggest we keep this issue open to gather more user feedback.
